src/logLevels.js

```diff
@@ -9,3 +9,10 @@ LogLevel[LogLevel.Debug = 4] = 'Debug'
 LogLevel[LogLevel.Trace = 5] = 'Trace'
 LogLevel[LogLevel.Silent = -Infinity] = 'Silent'
 LogLevel[LogLevel.Verbose = Infinity] = 'Verbose'
+
+export function normalizeLogLevel (input, types = {}, defaultLevel = 3) {
```

Comment:
Since literals are already preset, does it makes sense to directly use a reverse object map instead of accepting `types` param?

Reply:
I am not so sure about what `types` is supported to do. But since it's passed as an option, as guess it's something that user could provide custom log levels on it?

Reply:
Indeed you are right. Makes sense to ready level also from user customized options